src/foam/lib/json/AnyParser.java

```diff
@@ -18,6 +18,6 @@ public AnyParser() {
         new BooleanParser(),
         new LongParser(),
         new DoubleParser(),
-        new FObjectParser()));
+        new ExprParser()));
```

Comment:
The `ExprParser` consists of an `Alt` with the two options being an `FObjectParser` and a custom parser for properties (`{ class: "__Property__". forClass_: "xyz" }`). This is needed to ensure properties are also parsed and handled correctly (the `__Property__` class isn't available from `foam.lookup()`, so the `FObjectParser` crashes if left as is).